develop/TOOLS/DIS/Developing/Developing_Automation_scripts_as_Visual_Studio_solutions.md

```diff
@@ -30,16 +28,23 @@ As from DIS v2.31, similar to DataMiner protocols, it is also possible to develo
 > [!TIP]
 > For a video course on creating Visual Studio Automation script solutions, see [DataMiner Automation](https://community.dataminer.services/courses/dataminer-automation/) on DataMiner Dojo.
 
-## Creating an Automation Script solution
+## Creating an Automation script solution
+
+To create a new Automation script solution containing an initial Automation script, do the following:
 
-To create a new Automation script solution containing one dummy Automation script, do the following:
+1. Select *File \> New \> Project*.
+1. In the *Create a new project* pop-up window, select *DataMiner Automation Script Solution*.
+1. Enter the name and location of the solution, then click *Next*.
+1. Provide the name of the Automaton script and the author, and click *Create*.
 
-1. Select *File \> New \> DataMiner Automation Script Solution...*
+In Visual Studio 2019, perform the following steps:
```

Comment:
Which version of Visual Studio should be used when following the previous steps? If we offer two separate instructions, we should clearly clarify when to follow which steps.

Reply:
The other instructions apply to VS2022. The latest versoin of DIS only supports VS2019 and VS2022.